README.md

```diff
@@ -1 +1,28 @@
-# frontend-functional-tests
\ No newline at end of file
+Functional testing for frontend
+
+## Usage
+
+To run the tests, use the following command
+```
+node index.js
+```
+
+All results will be saved in `status.csv` with the following headers:
+`id,name,page_status,page_title,dataset_title,readme,csv_links,csv_preview_links,json_links,zip_links,datapackage_json,total_load_time,script_laod_time,layout_load_time,tables,graphs`
+
+If you want to add datasets to test, add dataset into dictionary in `index.js`
```

Comment:
Can't we load this from csv or some other document? Would be good to implement in future. We could test all core packages in that way

Reply:
We can create another csv file, and get from there, it is worth it?

Reply:
@zelima we can do it later, according to the issue we only need to test couple datasets: https://github.com/datahq/pm/issues/42

Reply:
@Mikanebu Let's open issue for that and do it later